### DISCUSSION TITLE
这么高的帧率有什么用？

### DISCUSSION THREAD

**Discussion by wuliaodexiaoluo** [Meloong-Git/PCL#3753] [讨论]
[21:24:36.341] [Report] FPS 56, 动画 0, 下载中 0（0 B/s）
这几乎接近 60 FPS 的超高动画帧率真的有必要吗？
我觉得 45 FPS 可能就很流畅了
长到卡顿的字符的罪魁祸首

**Comment by SALTWOOD** [comment]
你可以提一个 optimization，要求龙猫做一个限制的办法，或者直接去高级选项，把动画调为关闭。但是这也只是你觉得，有些人觉得帧率高些流畅就喜欢高帧率，你有啥办法？

**Reply by wuliaodexiaoluo** [reply]
6
